lib/ex_unit/lib/ex_unit/capture_server.ex

```diff
@@ -122,4 +169,18 @@ defmodule ExUnit.CaptureServer do
       Logger.add_backend(:console, flush: true)
     end
   end
+
+  defp device_by_ref(devices, ref) do
+    Enum.find(devices, fn {_, device} -> offset_by_ref(device, ref) end)
```

Comment:
```suggestion
    Enum.find(devices, fn {_, device} -> Map.has_key?(devices, ref) end)
```

offset_by_ref and refs_equals can be fully removed now. :)

Reply:
I think it should be:

```suggestion
    Enum.find(devices, fn {_, device} -> Map.has_key?(device.refs, ref) end)
```